### PR TITLE
Fix contrailCNI active status

### PR DESCRIPTION
**Pull request by Dysproz**
Quick fix for ContrailCNI active status which was not set to true after job complete.
Also removed unnecessary label for ContrailCNI from manager.

### CODE REVIEW

File: pkg/controller/manager/manager_controller.go [hunk 1]
```diff
@@ -674,9 +674,6 @@ func (r *ReconcileManager) processContrailCNIs(manager *v1alpha1.Manager) error
 			oldContrailCNI.ObjectMeta = v1.ObjectMeta{
 				Namespace: manager.Namespace,
 				Name:      *existingContrailCNI.Name,
-				Labels: map[string]string{
-					"contrail_cluster": "cluster1",
-				},
```
Comment (psykulsk) [inline pkg/controller/manager/manager_controller.go:679]: Now it will delete any ContrailCNI object in the namespace. Shouldn't it delete only the one in the same "contrail_cluster" as the current Manager?

**Review by psykulsk — CHANGES_REQUESTED** [review]

File: pkg/controller/manager/manager_controller.go (hunk 1 above)
Comment (Dysproz) [reply pkg/controller/manager/manager_controller.go]: Done - for all resources

**Review by psykulsk — APPROVED** [review]

File: pkg/controller/manager/manager_controller.go [hunk 2]
```diff
@@ -381,6 +387,9 @@ func (r *ReconcileManager) processWebui(manager *v1alpha1.Manager, replicas *int
 			oldWebUI.ObjectMeta = v1.ObjectMeta{
 				Namespace: manager.Namespace,
 				Name:      *manager.Status.Webui.Name,
+				Labels: map[string]string{
```
Comment (kaweue) [inline pkg/controller/manager/manager_controller.go:390]: Why is this required?

Comment (Dysproz) [reply pkg/controller/manager/manager_controller.go:390]: because manager adds label with its name to created resources which points to this manager as owner. So in case there would be other WebUI created by e.g. *cluster2* manager and we process this Delete without labels from *cluster1* manager then it'll delete *cluster2's* WebUI. Labels will protect resources created by other managers

Comment (kaweue) [reply pkg/controller/manager/manager_controller.go:390]: smart

**Review by kaweue — APPROVED** [review]
